WgServerforWindows/Models/ServerConfiguration.cs

```diff
@@ -34,25 +34,27 @@ public ServerConfiguration()
             {
                 Validate = obj =>
                 {
-                    string result = default;
-
-                    if (IPNetwork.TryParse(obj.Value, out _) == false)
+                    // Multiple server addresses are supported, so validate all of them
+                    foreach (string address in obj.Value.Split(new[] { ',' }).Select(a => a.Trim()))
                     {
-                        result = Resources.NetworkAddressValidationError;
-                    }
-                    else // TryParse succeeded
-                    {
-                        // IPNetwork.TryParse recognizes single IP addresses as CIDR (with 8 mask).
-                        // This is not good, because we want an explicit CIDR for the server.
-                        // Therefore, if IPNetwork.TryParse succeeds, and IPAddress.TryParse also succeeds, we have a problem.
-                        if (IPAddress.TryParse(obj.Value, out _))
+                        if (IPNetwork.TryParse(address, out _) == false)
                         {
-                            // This is just a regular address. We want CIDR.
-                            result = Resources.NetworkAddressValidationError;
+                            return Resources.NetworkAddressValidationError;
+                        }
+                        else // TryParse succeeded
+                        {
+                            // IPNetwork.TryParse recognizes single IP addresses as CIDR (with 8 mask).
+                            // This is not good, because we want an explicit CIDR for the server.
+                            // Therefore, if IPNetwork.TryParse succeeds, and IPAddress.TryParse also succeeds, we have a problem.
+                            if (IPAddress.TryParse(address, out _))
+                            {
+                                // This is just a regular address. We want CIDR.
+                                return Resources.NetworkAddressValidationError;
+                            }
                         }

```

Comment:
```suggestion
                        // IPNetwork.TryParse recognizes single IP addresses as CIDR (with 8 mask).
                        // This is not good, because we want an explicit CIDR for the server.
                        // Therefore, if IPNetwork.TryParse succeeds, and IPAddress.TryParse also succeeds, we have a problem.
                        if (IPAddress.TryParse(address, out _))
                        {
                            // This is just a regular address. We want CIDR.
                            return Resources.NetworkAddressValidationError;
                        }
```
What do you think about removing the `else` block, as it's returning inside the `if` block, and having or not having the `else` block should behave the same, shouldn't it?

Reply:
Agreed. Sometimes having an explicit `else` helps the reader to understand the logical flow, but I'm good with your suggestion here. I also fixed a `Split` in this file.